dhcpv4/dhcpv4.go

```diff
@@ -133,8 +139,8 @@ func GenerateTransactionID() (TransactionID, error) {
 }
 
 // New creates a new DHCPv4 structure and fill it up with default values. It
-// won't be a valid DHCPv4 message so you will need to adjust its fields.
-// See also NewDiscovery, NewRequest, NewAcknowledge, NewInform and NewRelease.
+// won't be a valid DHCPv4 message so you will need to adjust its fields. See
+// also NewDiscovery, NewRequest, NewAcknowledge, NewInform and NewRelease.
 func New(modifiers ...Modifier) (*DHCPv4, error) {
```

Comment:
can you make New be a wrapper of NewWithContext as well? just so changes to one aren't forgotten in the other.

Reply:
Done with [e0b42be](https://github.com/insomniacslk/dhcp/pull/555/commits/e0b42bed27c5d6babfb99dc400d5c924575a13ed).